### PR TITLE
Upper ECAM decimal font size change

### PR DESCRIPTION
**Pull request by RichardPilbery**
## Summary of Changes

Mainly cosmetic changes to match the reference screenshot. Throttle position indicator has been nudged up slightly to match the N1 cursor starting position.
Numbers after the decimal point on the N1 and N2 gauges are slightly smaller.

## Screenshots (if necessary)

decimal-fix-v2

## References

96678268-4b415a00-1369-11eb-8d34-0c1623962be7

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build.py will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, slick on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Comment by Benjozork** [comment]
It seems like the decimal IRL is a bit lower, with its baseline matching the whole number.

**Comment by beheh** [comment]
Agreed, N2 looks correct, N1 looks a little too high. Even though the fonts don't match you can see in the reference photo that the bottom part of the numbers are aligned.

**Comment by stevex77** [comment]
Great work!

Based on Reference images, the same should be for number and percentage character below thrust mode (upper-right corner).

I also noticed that left part of slats/flaps symbol looks slightly different - most left edge shouldn't be strictly versical, rather the symbol shoudl be more of "diamantish" shape. :)

Also the box around the EGT numeric value seems to be slight longer to the left.

**Comment by RichardPilbery** [comment]
decimal-fix-v3

**Comment by stevex77** [comment]
Now I've got a feeling that decimals for N1 and N2 are slightly below baseline (let's say one pixed on your screenshot) derived from whole numbers in front of them.
For upper right thrust mode value and percentage it's perfectly aligned.

Upper_ECAM

**Comment by RichardPilbery** [comment]
@Benjozork I wonder if the numbers are the decimal point are actually meant to be level with the whole number. I straightened the original image:

Screenshot 2020-10-19 at 06 21 44


Screenshot 2020-10-19 at 06 21 54

@stevex77 Are you saying that actually the numbers should all be level?

**Comment by stevex77** [comment]
@RichardPilbery Yes, I believe so. See for example this video (from 2:10): https://www.youtube.com/watch?v=aANO8Mka_WA
You can clearly see that bootom of all numbers should be in line.

**Comment by RichardPilbery** [comment]
@stevex77 Thanks. Unfortunately, this is not an A320neo (getting quality images/videos for that plane is a bit difficult) and @Benjozork is a bit of a stickler for getting the correct variant! Let's see what he says.

**Comment by Benjozork** [comment]
Hey, that's the only way we can match payware 😆

I'll see what I can find and get back to you

**Comment by stevex77** [comment]
OK, here is the neo, at 0:45, very crisp view of upper ECAM: https://www.youtube.com/watch?v=cv41hm4JEB8 ;)

**Comment by RichardPilbery** [comment]
Cool thanks. The N1 gauge is tricky to get positioned just right as I can only adjust the Y value in integer percentages. I might be able to cheat it by changing the font size of the decimal number slightly. Early tests resulted in it either being just a smidge too high or low, but if we're going with all numbers level, I can experiment some more.
Just let me know...we've got a few extra days before the next update.

**Comment by Benjozork** [comment]
image

**Comment by stevex77** [comment]
@Benjozork Great video of startup sequence you've found! :)

**Comment by RichardPilbery** [comment]
N2 looks level in this shot, thanks @stevex77 : https://youtu.be/cv41hm4JEB8?t=45


Screenshot 2020-10-27 at 18 25 06



Screenshot 2020-10-27 at 18 23 32

**Comment by RichardPilbery** [comment]
@stevex77 That's the one. I took stills from. Shame they are using the Pratt & Whitney engines.

**Comment by RichardPilbery** [comment]
Okay, I'm going all numbers level unless I hear otherwise, @Benjozork

**Comment by stevex77** [comment]
@RichardPilbery Yes, shame to them! :D

**Comment by Benjozork** [comment]
> 
> 
> Okay, I'm going all numbers level unless I hear otherwise, @Benjozork

LGTM

**Comment by RichardPilbery** [comment]
n2-closeup

N1-closeup


decimal-fix-v4

**Comment by stevex77** [comment]
Great work, thanks! Now it's perfect.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_ECAMGauge.js [hunk 1]
```diff
@@ -334,6 +346,8 @@ var A320_Neo_ECAM_Common;
                 this.currentValueText.setAttribute("class", style);
                 if (!this.isActive) {
                     this.currentValueText.textContent = "XX";
+                    this.currentValueTextdecimal.textContent = "";
+
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_ECAMGauge.js:350]:
```suggestion
```

**Review by Benjozork — APPROVED** [review]

**Comment by RichardPilbery** [comment]
Hey @atcopilot thanks for taking a look at this. It's really hard to be entirely sure as there seems to be colour leeching from other items on the screen. However, now you've pointed it out, I think you might be right. Completely agree with the decimal point position. What do you think @Benjozork ? I'll need to create a new text element to position the decimal point anyway, which would mean it would be possible to change the colour too.

**Comment by RichardPilbery** [comment]
Another reference here: https://youtu.be/aAvQXMbiodw?t=20

a320neo-upper-ecam

**Comment by stevex77** [comment]
Agree with dot position slightly more to the left.
Disagree with white dot color - shold be the same green as numbers, see this (N1 values decimal points are clearly green):
ECAM

**Comment by Benjozork** [comment]
Seems like it varies based on the display version ... Let's keep it green for now

**Comment by RichardPilbery** [comment]
decimal-fix-v4

**Comment by St54Kevin** [comment]
Just tested. Looks very nice good job. The things you did look like IRL. LGTM

**Comment by St54Kevin** [comment]
Nevermind. That laggin behind is also in the master.
LGTM

**Comment by danbw9** [comment]
LGTM. Decimal point fixed. Checked versus above references and those found on YT. 

I confirm the lagging behind has been there since the start.

**Comment by Benjozork** [comment]
Small enough to merge.

**Comment by Benjozork** [comment]
@RichardPilbery Please add a changelog next time.

**Pull request merged**